core/src/client/rpc.rs

```diff
@@ -84,6 +84,8 @@ trait EthRpc<TX: TransactionResponse + RpcObject, TXR: RpcObject, R: ReceiptResp
     async fn gas_price(&self) -> Result<U256, ErrorObjectOwned>;
     #[method(name = "maxPriorityFeePerGas")]
     async fn max_priority_fee_per_gas(&self) -> Result<U256, ErrorObjectOwned>;
+    #[method(name = "getBlobBaseFee")]
```

Comment:
As per the [spec](https://ethereum.github.io/execution-apis/api-documentation/), it is `eth_blobBaseFee` so the `get` is not required.